src/network/networkprotocol.h

```diff
@@ -573,6 +573,8 @@ enum ToClientCommand
 		v3f1000 world_pos
 		v2s32 size
 		s16 z_index
+		u32 len3
+		u8[len3] text2
```

Comment:
This reminds me some of these are inaccurate, `len`, `len2` and `len3` are u16 not u32